src/app/modules/auth/classic-auth/classic-auth.controller.ts

```diff
@@ -44,24 +46,23 @@ export class ClassicAuthController {
   async activate (
     @Res () response: Response,
     @Param ('token', ParseUUIDPipe) token: string,
+    @Req () request: Request,
   ) {
     response
       .status (HttpStatus.OK)
-      .send (await this.classicAuthService.activate (token));
+      .send (await this.classicAuthService.activate (token, request));
   }
 
-  @ApiOperation ({summary: 'Resend activation email(---! needs to be implemented)'})
+  @ApiOperation ({summary: 'Resend activation email'})
   @Post ('activate/resend')
-  resendActivationEmail () {
-    // Resend activation email.
-    // Payload should contain email address
-    // {
-    //   "email": "email@domain.com"
-    // }
-    // after sending email, return success message
-    // if email not found, return also success message to prevent email enumeration attack.
-    // The message "If the email is registered, an activation email will be sent" should be returned
-    return 'resendActivationEmail';
+  async resendActivationEmail (
+      @Res () response: Response,
+      @Body() classicAuthActivateResendPayloadDto: ClassicAuthActivateResendPayloadDto,
+      @Req() request: Request
+  ) {
+    return response
+      .status (HttpStatus.OK)
+      .send (await this.classicAuthService.resendActivationEmail(classicAuthActivateResendPayloadDto, request));
```

Comment:
Is not necessary to pass all request object. Pass just a part of request that you need